### PR TITLE
feat(orc8r): Enable / Disable 5G Feature-set

### PR DESCRIPTION
**Pull request by Abhishek-R15**
**Summary**
feat(orc8r): Enable / Disable 5G Feature-set

This PR is the extension of feat(orc8r): Enable / Disable 5G Standalone for Pipelined and SessionD #8541

Description : "enable5g_features" key added for SubscriberDB, Pipelined and Sessiond, and modified "enable_converged_core" with "enable5g_features" on MME
Request Type : GET/PUT
API url : "/lte/{network_id}/cellular/epc"

**Test Plan**
Tested with swagger API
Update in config Files. (/var/opt/magma/configs/gateway.mconfig)

**Additional Information**
Please refer below PR's for more changes
  1. #9016 - Sessiond related changes
  2. #9017 - pipelined related changes
  3. #9087 - MME and Subscriberdb changes

Signed-off-by: Abhishek R abhishek.r@wavelabs.ai

### CODE REVIEW

**Comment by Abhishek-R15** [comment]
image
image
/var/opt/magma/configs/gateway_mconfig
image
image

**Review by hcgatewood — CHANGES_REQUESTED** [review]
Please add a clearer, fleshed-out summary, otherwise I have no way to evaluate the content of this PR

**Comment by Abhishek-R15** [comment]
@hcgatewood Updated the summary, please review. Thanks!

**Comment by Abhishek-R15** [comment]
> Please add a clearer, fleshed-out summary, otherwise I have no way to evaluate the content of this PR

Updated the summary, please review the PR. Thanks!

File: lte/cloud/go/services/lte/servicers/builder_servicer_test.go [hunk 1]
```diff
@@ -1622,3 +1682,140 @@ func setEpcNetworkServiceAreaMap(nw *configurator.Network, serviceAreaMaps map[s
...
+		},
+		"liagentd": &lte_mconfig.LIAgentD{
+			LogLevel: protos.LogLevel_INFO,
+		},
+	}
+
+	actual, err := buildNonFederated(&nw, &graph, "gw1")
+	assert.NoError(t, err)
+	assert.Equal(t, expected, actual)
+}
```
Comment (uri200) [inline lte/cloud/go/services/lte/servicers/builder_servicer_test.go]: please, can you see if you can include this test on the main test where all the parameters are checked?

being 4g or 5g is agnostic to the config file

Comment (Abhishek-R15) [reply lte/cloud/go/services/lte/servicers/builder_servicer_test.go]: Hi @uri200 I have included the mentioned test into the main test, please review

**Review by uri200 — APPROVED** [review]
test looks good now

**Comment by uri200** [comment]
@andreilee @hcgatewood  can you take a look at nms part please

**Review by andreilee — APPROVED** [review]
Stamping for generated NMS changes

**Review by uri200 — APPROVED** [review]

**Review by hcgatewood — APPROVED** [review]
Deferring to @ulaskozat since this is domain-specific functionality

**Pull request merged**